src/libs/nav2_util/src/pf/pf.c

```diff
@@ -216,7 +216,7 @@ int pf_update_converged(pf_t *pf)
   int i;
   pf_sample_set_t *set;
   pf_sample_t *sample;
-  double total;
+  /*double total*/;
```

Comment:
This should be just deleted. It's not used and also not a parameter.

Reply:
Agreed.